app/screens/phone-auth-screen/phone-auth.tsx

```diff
@@ -93,15 +98,14 @@ type WelcomePhoneInputScreenProps = {
 export const WelcomePhoneInputScreen: ScreenType = ({
   navigation,
 }: WelcomePhoneInputScreenProps) => {
-  const [requestPhoneCode, { loading }] = useMutation(REQUEST_PHONE_CODE, {
+  const [requestPhoneCode, { loading }] = useMutation(REQUEST_AUTH_CODE, {
     fetchPolicy: "no-cache",
   })
 
   const inputRef = useRef<PhoneInput | null>()
 
   const send = async () => {
```

Comment:
What do you think about including `useCallback` hook here? Can be part of a separate code cleanup PR.

Reply:
Is it not already? https://github.com/apollographql/react-apollo/issues/3260

Reply:
I'm referring to the `send` function.

```suggestion
  const send = useCallback(async () => {
```

Reply:
Oh, then yes. In general we aren't using enough useCallbacks throughout the app. I can add it now.